twilio/Module.md

```diff
@@ -1,337 +1,92 @@
-    import ballerina/log;
-    import ballerinax/twilio;
-
-    configurable string accountSId = ?;
-    configurable string authToken = ?;
-
-    public function main() {
-        //Twilio client configuration
-        twilio:TwilioConfiguration twilioConfig = {
-            accountSId: accountSId,
-            authToken: authToken
-        };
-
-        //Twilio client
-        twilio:Client twilioClient = new (twilioConfig);
-
-        //Get account detail remote function is called by the twilio client
-        var details = twilioClient->getAccountDetails();
-
-        //Response is printed as log messages
-        if (details is twilio:Account) {
-            log:printInfo("Account Detail: " + details.toString());
-        } else {
-            log:printInfo(details.message());
-        }
-    }
-```
-
-### Send an SMS
-This section shows how to use the connector to send an SMS. You will need a verfied phone number if you are using a trial account to send the message from your twilio phone number. if the SMS is sent successfully it will provides SMSResponse record with details of the SMS otherwise it will provide the error occured.
-Sample is available at: samples/client samples/sendSMS.bal
-```ballerina
-import ballerina/log;
-import ballerinax/twilio;
-
-configurable string fromMobile = ?;
-configurable string toMobile = ?;
 configurable string accountSId = ?;
 configurable string authToken = ?;
-configurable string message = "Wso2-Test-SMS-Message";
 
+twilio:TwilioConfiguration twilioConfig = {
+    accountSId: accountSId,
+    authToken: authToken
+};
 
-public function main() {
-    //Twilio client configuration
-    twilio:TwilioConfiguration twilioConfig = {
-        accountSId: accountSId,
-        authToken: authToken
-    };
+twilio:Client twilioClient = new (twilioConfig);
+```
 
-    //Twilio client
-    twilio:Client twilioClient = new (twilioConfig);
+### Step 3 - Invoke  connector operation
+1. Invoking connector operation using the client
+```ballerina
+public function main() returns error? {
+    twilio:Account response = check twilioClient->getAccountDetails();
+}
+```
+2. Use `bal run` command to compile and run the Ballerina program.
 
-    //Send SMS remote function is called by the twilio client
-    var details = twilioClient->sendSms(fromMobile, toMobile, message);
+## Quick reference
+Code snippets of some frequently used functions: 
 
-    //Response is printed as log messages
-    if (details is twilio:SmsResponse) {
-        log:printInfo("SMS_SID: " + details.sid.toString() + ", Body: " + details.body.toString());
-    } else {
-        log:printInfo(details.message());
-    }
-}
+* Send an SMS
+This section shows how to use the connector to send an SMS. You will need a verfied phone number if you are using a trial account to send the message from your twilio phone number. if the SMS is sent successfully it will provides SMSResponse record with details of the SMS otherwise it will provide the error occurred.
```

Comment:
Suggestion: The following snippet shows how to send an SMS via the connector. If you are using a trial account, you need a verified phone number to send the message from your Twilio phone number. If the SMS is successfully sent, it. returns the `SMSResponse` record with details of the SMS. If the SMS was not sent, it returns the error that occurred."